addon/components/cf-field/input/float.js

```diff
@@ -9,11 +9,14 @@ import Component from "@ember/component";
 export default Component.extend({
   tagName: "input",
   classNames: ["uk-input"],
-  classNameBindings: ["field.isInvalid:uk-form-danger"],
+  classNameBindings: [
+    "field.isInvalid:uk-form-danger",
+    "disabled:disabled"
```

Comment:
Where is the styles of `.disabled`? Also, isn't there a class `uk-disabled`?

Reply:
Good catch. It's missing from this MR. I directly put it into the project code.

```scss
.uk-input.disabled,
.uk-textarea.disabled {
  background-color: $form-disabled-background;
  color: $form-disabled-color;
  @if(mixin-exists(hook-form-disabled)) {@include hook-form-disabled();}
}
```

That should be included in the PR. I'll check why I didn't name this `uk-disabled`. I guess there's no such class in UIkit as they - IIRC - style through the element's attribute. I'll keep you posted.

Reply:
Yeah. They style through the attribute.
https://github.com/uikit/uikit/blob/5e0b52085fae878b08fc38107b34fa1629f8b22c/src/scss/components/form.scss#L290

I'll create a PR.

Reply:
https://github.com/uikit/uikit/pull/3990

Reply:
I'm not sure if this PR will be accepted in the near future. Would it be a good idea to put @fkm your snipped above in the `ember-caluma` codebase?

Reply:
Bump :point_up:

Reply:
I agree that this PR is not going to be merged (any time soon). I'm ok with putting the styles into our code base.

Reply:
@fkm can you update the PR?

Reply:
Ok

Reply:
Done